browser_patches/webkit/archive.sh

```diff
@@ -28,6 +28,11 @@ if ! [[ -d $(dirname "$ZIP_PATH") ]]; then
   exit 1
 fi
 
+IS_UNIVERSAL=""
+if [[ $2 == "--universal" ]]; then
```

Comment:
The code below assumes that this flag can be combined with others hence doesn't not necessarily come as second argument. Let's move argument parsing into a single location?